Software/app/src/main/java/org/foi/emp/collegecoursemanager/DodavanjeKolegija.java

```diff
@@ -0,0 +1,101 @@
+package org.foi.emp.collegecoursemanager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.Display;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import org.foi.emp.core.Database.Database;
+import org.foi.emp.core.Entities.ElementModelaPracenja;
+import org.foi.emp.core.Entities.ElementiNaModeluPracenja;
+import org.foi.emp.core.Entities.Kolegij;
+import org.foi.emp.core.Entities.ModelPracenja;
+
+public class DodavanjeKolegija extends AppCompatActivity {
+    Button spremi;
+    EditText nazivKolegija;
+    EditText modelPracenja;
+    EditText kolokvij1, kolokvij2, aktivnost, labosi;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dodavanje_kolegija);
+        spremi = findViewById(R.id.btnSpremiKolegij);
+        nazivKolegija = findViewById(R.id.txtNazivKolegija);
+        modelPracenja = findViewById(R.id.txtModelPracenja);
+        kolokvij1 = findViewById(R.id.txtKolokvij1MoguciBodovi);
+        kolokvij2 = findViewById(R.id.txtKolokvij2MoguciBodovi);
+        aktivnost = findViewById(R.id.txtAktivnostMoguciBodovi);
+        labosi = findViewById(R.id.txtLabosiMoguciBrojBodova);
+        spremi.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if(!nazivKolegija.getText().toString().isEmpty()){
+                    final String naziv = nazivKolegija.getText().toString();
+                    String model = modelPracenja.getText().toString();
```

Comment:
`String model = this.modelPracenja.getText().toString();`

Apply the same practice further below.